packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -1487,6 +1488,20 @@ class ReviewSectionComp extends React.Component<FullProps, State> {
     )
   }
 
+  isLastNameFirst = () => {
+    const { registerForm, draft: declaration } = this.props
+    const fields = registerForm[declaration.event].sections.find((section) =>
+      declaration.event === Event.Birth
+        ? section.id === BirthSection.Child
+        : section.id === DeathSection.Deceased
```

Comment:
@Zangetsu101 lets do the same for Informant, Mother and Father

Reply:
If you reorder the fields to put the lastName before the firstName, the fullname on the review section for everyone (child, mother, father) shows accordingly. It's just the review header (where the name of the child or deceased shows up) wasn't being updated.
![image](https://user-images.githubusercontent.com/29002716/207865047-35891949-3627-47c9-a2a1-45b403062bcb.png)